src/token.jl

```diff
@@ -118,8 +118,6 @@ function sortedcomplement(of::SourceRange, set)
 end
 
 ⨳(sym::Symbol) = Expr(sym)
-⨳(sym::Symbol, args::LineNumberNode...) = Expr(sym::Symbol, args...)
```

Comment:
@KristofferC these my not be exercised in the tests but they may be used in the ASTInterpreter.jl source code, have you checked this?